### PR TITLE
Adding parameter timeout for waiting cluster up

### PR DESCRIPTION
**Pull request by aleksbykov**
If host is busy or slow, scylla in docker container start slowly
and tests in scylla_docker.py are failed by timeout which is 30
Addint to configuration and possiblity to customize this timeout:
- in yaml file scylla-docker.yaml 'timeupwait'
- adding parameter to ScyllaDocker

Addintionally, tests failed due error on parsing output:
> Op rate                   :   18,589 op/s  [READ: 18,589 op/s]
> Partition rate            :   18,589 pk/s  [READ: 18,589 pk/s]

with exception
_ERROR| Failed parsing stress results: Op rate:   11,779 op/s  [WRITE: 11,779 op/s], error: invalid literal for float(): 11,779_
Adding code to cut the ',' from parsed value.

### CODE REVIEW

**Comment by bentsi** [comment]
@amoskong please review

File: scylla-artifacts.py.data/scylla-docker.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Docker
 docker_image: scylladb/scylla-nightly:latest
 node_cnt: 2
+timeupwait: 30
```
Comment (amoskong) [inline scylla-artifacts.py.data/scylla-docker.yaml]: How about rename it to `start_timeout`?

File: scylla_docker.py [hunk 2]
```diff
@@ -159,7 +160,7 @@ def get_stress_results(stress_out):
                 try:
                     res = line.split(':')
                     key = res[0].strip()
-                    val = res[1].split()[0].strip() if key != 'Total operation time' else\
+                    val = res[1].split()[0].strip().replace(',', '') if key != 'Total operation time' else\
```
Comment (amoskong) [inline scylla_docker.py]: Let's split this change to another patch :-)

File: scylla_docker.py [hunk 3]
```diff
@@ -96,7 +97,7 @@ def wait_for_cluster_up(self):
         node_ips = [self.get_node_ip(name) for name in self.nodes]
         up_nodes = list()
         try_cnt = 0
-        while len(up_nodes) < len(node_ips) and try_cnt < 30:
+        while len(up_nodes) < len(node_ips) and try_cnt < self._timeupwait:
```
Comment (amoskong) [inline scylla_docker.py]: typo:  self.timeupwait

File: scylla_docker.py [hunk 4]
```diff
@@ -194,9 +196,10 @@ def setUp(self):
         """
         Update scylla image, create cluster(cleanup if exists)
         """
-        self.docker = ScyllaDocker(image=self.image, node_cnt=self.node_cnt)
+        self.docker = ScyllaDocker(image=self.image, node_cnt=self.node_cnt, timeupwait=self.timeupwait)
         self.docker.update_image()
         self._cleanup()
+        log.debug('Wait cluster timeup: {}'.format(self.timeupwait))
```
Comment (amoskong) [inline scylla_docker.py]: add time unit : seconds

File: scylla-artifacts.py.data/scylla-docker.yaml (hunk 1 above)
Comment (aleksbykov) [reply scylla-artifacts.py.data/scylla-docker.yaml]: Agree!

File: scylla_docker.py (hunk 2 above)
Comment (aleksbykov) [reply scylla_docker.py]: Split. Remove from this commit

**Comment by amoskong** [comment]
Hi @aleksbykov , I just prepare a new branch as example for you:
 https://github.com/scylladb/scylla-artifact-tests/commits/add-timeout-to-create-cluster-v2

It contains two patches(commits), one for up timeout, one for fix parse output.
more fix in two commits:
- fixed author to scylla email
- added Signed-off-by

**Comment by roydahan** [comment]
@amoskong / @aleksbykov please make sure that the jenkins job passes after this change.

**Comment by amoskong** [comment]
I just started a new job, @aleksbykov and I will watch it.
http://jenkins.cloudius-systems.com:8080/job/scylla-master/job/manual-and-scheduled-tests/job/docker-test/label=muninn/6/console

**Comment by amoskong** [comment]
The test job failed for other issue which isn't caused by this PR change.
I had fixed it by another PR #116 

A new test job (contains this PR) is PASS:
http://jenkins.cloudius-systems.com:8080/job/scylla-master/job/manual-and-scheduled-tests/job/docker-test/label=muninn/8/console

**Comment by amoskong** [comment]
Backported this PR to branch-3.0
